string state # state of the system in format for head display (e.g."Rolling Dice"). If the state is "Dice Read", headdisplay.py will display values and determine next move. 
int turn # turn number (total=13)
int roll # roll numer (either 1,2, or 3)
int dice1 # number of dots on dice1
int dice2 # number of dots on dice2
int dice3 # number of dots on dice3
int dice4 # number of dots on dice4
int dice5 # number of dots on dice5

